### DISCUSSION TITLE
The signal identifier in the Text event isn't recognized

### DISCUSSION THREAD

**Discussion by fire7side** [dialogic-godot/dialogic#755] [Q&A] [answered]
I'm trying to get a character to talk during the text rollout by sending a signal but it's ignored and printed as text.
[signal = test] Hello There.  I've tried using quotes and not.  The brackets aren't printed on screen for [br] so it seems to be ignoring it.  Any help would be appreciated.

**Comment by Jowan-Spooner** [comment, marked as answer]
What version are you using? It works for me on the current WIP version. It is not supported in the 1.3 release.

**Reply by Jowan-Spooner** [reply]
@fire7side Also I noticed that you have a space between the signal and the =. Try removing it if you are using 1.4 WIP.

**Reply by Jowan-Spooner** [reply]
Addition: I've just merged a pr that ignores all whitespace characters, so a space will no further prevent these commands from being found and executed. It will be part of the 1.4 release.

**Reply by fire7side** [reply]
I was using version 1.3, which was the problem, but I think adding the ignore for space is a good idea as I tend to type that way.  Working good now, so thanks for the info.
